apps/api-documenter/src/documenters/IConfigFile.ts

```diff
@@ -9,37 +9,45 @@ import { IYamlTocFile } from '../yaml/IYamlTocFile';
 export interface IConfigTableOfContents {
```

Comment:
>IConfigTableOfContents [](start = 17, length = 22)

Is there a schema file you need to update too?

Reply:
The only place I can think of it this file: https://github.com/microsoft/web-build-tools/blob/master/apps/api-documenter/src/schemas/api-documenter-template.json
@octogonz do I need to update this file with the changes I bring to `IConfigFile`? Where that template is being used?